great_expectations/render/util.py

```diff
@@ -209,3 +212,219 @@ def handle_strict_min_max(params: dict) -> (str, str):
+
+    In cases where the number of indices is too great (max 10 by default), the remaining values are
+    truncated and the column contains "..." in their place.
+
+                pk_1
+    giraffe     3
+    lion        4
+    zebra       5 6 8 ...
+
+    Args:
+
+        unexpected_index_list : all unexpected values and indices
+        partial_unexpected_counts : counts for unexpected values (max 20 by default)
+        unexpected_index_column_names:  in the case of defining ID/PK columns
+        unexpected_list: if we are using default Pandas output.
+
+    Returns:
+        pd.DataFrame that contains indices for unexpected values
+    """
+    if unexpected_index_column_names:
+        # if we have defined unexpected_index_column_names for ID/PK
+        unexpected_index_list_as_string: pd.DataFrame = pd.DataFrame(
+            unexpected_index_list, dtype="string"
+        )
+        unexpected_index_list_as_string = unexpected_index_list_as_string.fillna(
+            value="null"
+        )
+        domain_column_name: str = (
+            set(unexpected_index_list[0].keys())
+            .difference(set(unexpected_index_column_names))
+            .pop()
+        )
+    elif unexpected_list:
+        # if we are using default Pandas unexpected indices
+        unexpected_index_list_as_string = pd.DataFrame(
+            list(zip(unexpected_list, unexpected_index_list)),
+            columns=["Value", "Index"],
+            dtype="string",
+        )
+        unexpected_index_list_as_string = unexpected_index_list_as_string.fillna(
+            value="null"
+        )
+        domain_column_name: str = "Value"
+        unexpected_index_column_names: List[str] = ["Index"]
+    else:
+        return None
+
+    # unexpected indices are joined as list
+    all_unexpected_indices: pd.DataFrame = unexpected_index_list_as_string.groupby(
+        domain_column_name
+    ).agg(lambda y: list(y))
+
+    # filter all_unexpected_indices according to partial_unexpected_counts, since it has a maximum of 20 values by default
+    values_we_have_counts_for: List[str] = list(
+        pd.DataFrame(partial_unexpected_counts)["value"]
+    )
+    # list comprehension to replace None with 'null'
+    values_we_have_counts_for = [
+        "null" if val is None else val for val in values_we_have_counts_for
+    ]
+
+    filtered_unexpected_indices = all_unexpected_indices[
+        all_unexpected_indices.index.isin(values_we_have_counts_for)
+    ]
+
+    # applying function to every row in Pandas
+    # https://www.geeksforgeeks.org/apply-function-to-every-row-in-a-pandas-dataframe/
+    for column in unexpected_index_column_names:
+        filtered_unexpected_indices[column] = filtered_unexpected_indices[column].apply(
+            lambda row: filter_list_of_indices(row)
+        )
+
+    return filtered_unexpected_indices
+
+
+def filter_list_of_indices(indices: List[Union[int, str]], max_index: int = 10) -> str:
```

Comment:
What about changing the name to `truncate_list_of_indices`?

Reply:
👍🏼